app/repositories/supabase_repository.py

```
@@ -35,7 +35,7 @@ def add(self, device_id: str, business_id: str, biz_item_id: str) -> bool:
             return True
         except Exception as e:
             logger.error(f"Error adding favorite: {e}")
```

Ruff (0.15.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)